distributed/tests/test_worker.py

```diff
@@ -2859,14 +2865,21 @@ async def test_remove_replica_simple(c, s, a, b):
     while not all(len(s.tasks[f.key].who_has) == 1 for f in futs):
         await asyncio.sleep(0.01)
```

Comment:
This is now instantaneous. It should be moved immediately after _remove_replicas and without a wait